apps/okreads/browser/src/app/app.component.scss

```diff
@@ -14,7 +14,7 @@ h1 {
   position: sticky;
   top: 0;
   z-index: 100;
-  background-color: $pink-accent;
+  background-color: $pink-dark;
```

Comment:
What precipitated this change?

Reply:
accessibility contrast flagged in lighthouse report. (Reading List button)